rspirv/lift/storage.rs

```diff
@@ -56,4 +56,10 @@ impl<T> LiftStorage<T> {
         entry.insert(token);
         token
     }
+
+    pub(in crate::lift) fn lookup_token(
```

Comment:
What's the value of this over normal indexing? Also, this looks like it will panic on invalid code, which I'm not sure about.

Reply:
I think panicking is the behavior today for any lookup in the storage. It's not great, but we should address this separately.

Just indexing is not possible because `lookup` field is private. The idea is that `values` and `lookup` should always be in sync here, so we don't expose either directly.